### DISCUSSION TITLE
Changed pointer cursors

### DISCUSSION THREAD

**Discussion by 54m4d** [Daksh777/SpotifyNoPremium#10] [Q&A] [answered]
Hello Daksh,

I noticed the cursor in the latest changes of the theme changed from the default one <img src="https://img.icons8.com/metro/18/000000/cursor.png"/> to this one <img src="https://img.icons8.com/windows/20/000000/hand-cursor.png"/>

In the changelog i see something like  "**Update CSS for more accessibility**" what is it and how does it work?

**Comment by Daksh777** [comment, marked as answer]
Hey Fosam,

I've used the CSS property `cursor: pointer` to change the cursor <img src="https://img.icons8.com/ios/20/FFFFFF/cursor--v1.png" align="center"/>  to a pointer <img src="https://img.icons8.com/fluent-systems-regular/20/FFFFFF/hand-cursor.png" align="center"/> when the user hovers over a clickable element.
It's recommended to do it for visually impaired people and generally for accessibility (indicating that there is a clickable element and it's not just a part of the non-interactable part of the app).

**Reply by 54m4d** [reply]
Can i remove the pointer code from the theme to be able to use the default cursor because i prefer that?

**Reply by Daksh777** [reply]
Yes, you can just delete lines 33-57 in `user.css` and that should do.
